### PR TITLE
cmake: only include CajitaTargets when Cajita is on

### PR DESCRIPTION
**Pull request by junghans**
Found here https://github.com/ECP-copa/Cabana-fortran-examples/pull/6

### CODE REVIEW

**Comment by junghans** [comment]
@dalg24 jenkins is broken, some error with gpg ?!

**Comment by sslattery** [comment]
retest this please

**Comment by junghans** [comment]
The error is the same:
```
gpg: directory '/root/.gnupg' created
gpg: keybox '/root/.gnupg/pubring.kbx' created
gpg: keyserver receive failed: General error
```

**Comment by sslattery** [comment]
@dalg24 any thoughts on the Jenkins CUDA build issue?

**Comment by JDTruj2018** [comment]
@dalg24 @sslattery I ran into that same error when running `docker build .` on my Macbook. What fixed it was specifying a keyserver in the Dockerfile by changing line 24 from `gpg --recv-keys ${CMAKE_KEY} && \` to `gpg --keyserver keys.gnupg.net --recv-keys ${CMAKE_KEY} &&\`

**Comment by junghans** [comment]
Cool, that fixed it!

File: docker/Dockerfile [hunk 1]
```diff
@@ -21,7 +21,7 @@ RUN CMAKE_VERSION=3.13.4 && \
     wget --quiet ${CMAKE_URL}/${CMAKE_SHA256} && \
     wget --quiet ${CMAKE_URL}/${CMAKE_SHA256}.asc && \
     wget --quiet ${CMAKE_URL}/${CMAKE_SCRIPT} && \
-    gpg --recv-keys ${CMAKE_KEY} && \
+    gpg --keyserver keys.gnupg.net --recv-keys ${CMAKE_KEY} && \
```
Comment (dalg24) [inline docker/Dockerfile:24]: My experience is that retrieving keys from server will occasionally fail regardless.  On ArborX, at some point we got frustrated we the rate of failure and I [created a keydump](https://github.com/arborx/ArborX/blob/5d53ad63f82acca72279b105c3632569d75c7d78/docker/Dockerfile#L26-L32).  On Kokkos, we specify [yet another key server](https://github.com/kokkos/kokkos/blob/c836655e8236a57cf1522b966e5fa5777899d319/scripts/docker/Dockerfile.clang#L21) and so far we accept that we sometimes will need to relaunch CI builds.

**Pull request merged**